index.js

```diff
@@ -1,17 +1,50 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos > this.items.length){
+    throw new Error('OutOfBounds'); 
+  } else {
+    return this.items.indexOf(pos);
+  }
+  }
 
-  max() {}
+  max() {
+    if (this.items.length === 0){
+      throw new Error('EmptySortedList');
+    } else {
+      return Math.max(...this.items);
+    }
+  }
 
-  min() {}
+  min() {
+    if (this.items.length === 0){
+      throw new Error('EmptySortedList');
+    } else {
+      return Math.min(...this.items);
+    }
+  }
 
-  sum() {}
+  sum() {
+    return this.items.reduce((acc, value) => (acc += value), 0);
+  }
 
-  avg() {}
+  avg() {
+    if (this.items.length === 0){
+      throw new Error('EmptySortedList');
+    } else {
+      return this.items.reduce((acc, value) => (acc += value), 0) / this.items.length;
```

Comment:
Damn, Alejandro.
You were so close to perfection!
Im kidding, the only feedback is really that you could have reused the sum method and use this.length
```js
return this.sum() / this.length
```